File: 1848.py
lista = []
vezes = 0
soma = 0
while True:
    n = input()
    if n == 'caw caw':
        vezes += 1
        lista+=[soma]
        soma = 0
    else:
        if n[0] == '*' and n[1] == '-' and n[2] == '-':
            soma += 4
        elif n[0] == '-' and n[1] == '-' and n[2] == '*':
            soma += 1
        elif n[0] == '*' and n[1] == '*' and n[2] == '*':
            soma += 7
        elif n[0] == '*' and n[1] == '-' and n[2] == '*':
            soma += 5
        elif n[0] == '*' and n[1] == '*' and n[2] == '-':
            soma += 6
        elif n[0] == '-' and n[1] == '*' and n[2] == '*':
            soma += 3
    if vezes == 3:
        break
print(f'''{lista[0]}
{lista[1]}
{lista[2]}''')

File: 1848.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    string linha;
    int vezes = 0, soma = 0;
    while(true){
        getline(cin, linha);
        if(linha == "caw caw"){
            vezes += 1;
            cout << soma << endl;
            soma = 0;
        }else{
            if(linha[0] == '*' && linha[1] == '-' && linha[2] == '-')
                soma += 4;
            else if(linha[0] == '-' && linha[1] == '-' && linha[2] == '*')
                soma += 1;
            else if(linha[0] == '*' && linha[1] == '*' && linha[2] == '*')
                soma += 7;
            else if(linha[0] == '*' && linha[1] == '-' && linha[2] == '*')
                soma += 5;
            else if(linha[0] == '*' && linha[1] == '*' && linha[2] == '-')
                soma += 6;
            else if(linha[0] == '-' && linha[1] == '*' && linha[2] == '*')
                soma += 3;
        }if(vezes == 3)
            break;
    }
    return 0;
}
